tests/wpt/web-platform-tests/html/semantics/scripting-1/the-script-element/script-not-executed-after-shutdown.js

```diff
@@ -0,0 +1 @@
+// this script does nothing
```

Comment:
This script should call a function in the parent page which would make the test fail. Something like
`window.script_executed = t.unreached_func("script executed in removed iframe")` in the parent, and in the child: `script_executed = parent.script_executed;` then invoke `script_executed()` in this JS file.